CONTRIBUTING.md

```diff
@@ -283,6 +284,7 @@ Releasing to npm consists of the following phases:
     1. Merge the PR.
     1. Once merged, publish the release notes from `CHANGELOG.md` using [GitHub’s “draft new release tag” option](https://github.com/puppeteer/puppeteer/releases/new).
         - **NOTE**: tag names are prefixed with `'v'`, e.g. for version `1.4.0` the tag is `v1.4.0`.
+        - **NOTE**: in the case of Chromium roll, make sure to use `Chromium <version> (<revision>)` format, e.g. _"Chromium 90.0.4427.0 (r856583)"_ for version `v8.0.0.`.
```

Comment:
Same here. This should not be a manual step. Instead, we should state that roll commit messages should follow this format.

Reply:
I've removed this line totally as it is not a relevant warning if the commit is responsible for the autogenerated content. Thanks for the notice!